tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/handler/__bindingIdCamelCase__Handler.java

```diff
@@ -20,28 +23,28 @@
 /**
  * The {@link ${bindingIdCamelCase}Handler} is responsible for handling commands, which are
  * sent to one of the channels.
- * 
+ *
  * @author ${author} - Initial contribution
  */
 public class ${bindingIdCamelCase}Handler extends BaseThingHandler {
 
-    private Logger logger = LoggerFactory.getLogger(${bindingIdCamelCase}Handler.class);
+    private final Logger logger = LoggerFactory.getLogger(${bindingIdCamelCase}Handler.class);
```

Comment:
We could think about using `LoggerFactory.getLogger(getClass());`

The main difference is if you create a sub-class.